apps/auth/oauth2.py

```
@@ -0,0 +1,27 @@
+from authlib.integrations.starlette_client import OAuth, OAuthError
+from main import app
```

Ruff (0.8.2)

`main.app` imported but unused

Remove unused import: `main.app`

(F401)